docs/howto/quickstart.md

```diff
@@ -0,0 +1,281 @@
+# Quick start application deployment using ASO
+
+In this guide we will:
+ - Create a Service Principal
+ - Create an Azure Container Registry
+ - Create an AKS cluster with the ACR attached
+ - Deploy ASO
+ - Deploy an Azure Votes app that requires Azure SQL Server using ASO
+
+## Set up the Kubernetes environment and deploy ASO
+
+ ### Requirements
+- az cli ([install instructinos](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
+- kubectl cli ([install instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/))
+- helm cli ([install instructions](https://helm.sh/docs/intro/install/))
+- jq [install instructions](https://stedolan.github.io/jq/download/)
+
+### Prepare credentials and variables
+
+Set these variables now as they will help with copy/paste of commands later.
+
+>Only the first 3 environment variables need to be changed before pasting.
+
+```
+VOTES_APP_INSTANCE_NAME="<alias>" # only alpha numeric
+TENANT_ID=""
+SUBSCRIPTION_ID=""
```

Comment:
Can we add instructions on how people can get the values for these variables?

Reply:
Please also add some links that describe what those `ID` are

Reply:
can do!

Reply:
I'm adding the instructions for finding them. I'm not sure a quick start for someone completely unfamiliar to Azure is entirely in scope here. Plus, I don't know if MS has a definition listed anywhere.